Documentation/FirstProject/CreateSitePackage.rst

```diff
@@ -0,0 +1,46 @@
+..  include:: /Includes.rst.txt
+
+..  _creating-a-site-package:
+
+=======================
+Creating a site package
+=======================
+
+The next step is to create a site package. In TYPO3, a site package is a
+structured package that organizes the layout, design, and essential
+configuration settings for your website. By using a site package, you can
+manage all customizations in one place, which keeps your changes organized and
+makes your project easier to maintain, even through TYPO3 updates.
+
+..  _how-to-create-a-site-package:
+
+How to create a site package
+============================
+
+To create a site package, you have two main options:
+
+*   Manual creation: For full control over your project's setup, you can follow
+    the detailed instructions int the
+    :ref:`TYPO3 site package tutorial <t3sitepackage:start>`.
+
+*   Using the `Sitepackage-Builder <https://www.sitepackagebuilder.com/>`__:
+    If you are looking for a faster start, the Sitepackage-Builder is a
+    handy tool. You can choose whether your site package should be based on
+    the `Bootstrap Package <https://www.bootstrap-package.com/>`__ or
+    :ref:`fluid_styled_content <typo3/cms-fluid-styled-content:introduction>`.
+
+
+.. _pros-and-cons-of-sitepackage-builder:
+
+Pros and cons of using the Sitepackage-Builder
+==============================================
+
+Pros:
+
+*   The Sitepackage-Builder is particularly useful for beginners or projects
+    that need a quick start.
+Cons:
```

Comment:
Don't know, if it needs extra space here, but looks ugly to me.

```suggestion

Cons:
```

Reply:
yes Extra spaces after lists is needed